tasker/task/views.py

```diff
@@ -31,10 +32,24 @@ def task_detail(id):
 
     return render_template("task/task-detail.html", title="Task Detail", user=user, task=task)
 
-@bp.route('/add_task')
-#@login_required
+@bp.route('/add_task', methods=['GET', 'POST'])
+@login_required
 def add_task():
-    return render_template('task/add-task.html', title="Create Task")
+    form = TaskForm()
+    if form.validate_on_submit():
+        user_tz = timezone(current_user.timezone)
+        due_date = user_tz.localize(datetime.combine(form.due_date.data, datetime.min.time()))
+        task = Task.create_task(
+            form.name.data, form.description.data,
+            TaskStatus.Pending, due_date
+        )
+        task.hour = form.hour.data
```

Comment:
You can remove this line.  There is no hour field on the Task model.

Reply:
Removed line 46